integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -921,3 +923,28 @@ private[sql] case class CleanFiles(
     Seq.empty
   }
 }
+
+private[sql] case class ShowPartitions(tableIdentifier: TableIdentifier)
+  extends RunnableCommand {
+
+  override val output: Seq[Attribute] = {
+    AttributeReference("partition", StringType, nullable = false)() :: Nil
+  }
+
+  def run(sqlContext: SQLContext): Seq[Row] = {
```

Comment:
It seems this command is implemented twice in spark and spark2, can you move it to spark-common module?